language/vm/src/file_format.rs

```diff
@@ -328,6 +328,14 @@ pub struct FunctionDefinition {
     pub function: FunctionHandleIndex,
     /// Flags for this function (private, public, native, etc.)
     pub flags: u8,
+    /// List of nominal resources (local to this module) that the procedure might access
+    /// Either through: BorrowGlobal, MoveFrom, or transitively through another procedure
+    ///
+    /// Not in the signature as it is not needed outside of the declare modules
+    ///
+    /// Note, there is no LocalsSignatureIndex with each struct definition index, as global
```

Comment:
What is the connection between LocalsSignatureIndex and type arguments?

Reply:
It should really be renamed. `LocalsSignatureIndex` is used by generics as a way of storing a vector of signature tokens in a pool